### ISSUE TITLE
compatibility with Emulation.setVirtualTimePolicy

### ISSUE THREAD

**Issue by dreyks** [rubycdp/cuprite#264]
I'm trying to mock time on the frontend, for this I'm mocking time on the backend using ActiveSupport's travel_to and then trying to make it propagate to the browser

```rb
travel_to Time.zone.local(2024, 'jan', 1)
page.driver.browser.page.command(
  'Emulation.setVirtualTimePolicy',
  policy: 'pause',
  initialVirtualTime: Time.zone.now.to_i
)
p page.driver.browser.evaluate('(new Date()).toString()')
```

this does work returning `Mon Jan 01 2024 00:00:00` but then i get an error
> Ferrum::PendingConnectionsError: Request to http://127.0.0.1:55665/ reached server, but there are still pending connections: http://127.0.0.1:55665/

looks like something doesn't work well with virtual time

according to the [docs](https://chromedevtools.github.io/devtools-protocol/tot/Emulation/#type-VirtualTimePolicy) `policy: 'pause'` means that browser is not allowed to fast-forward the time when it's out of work
if i pass `policy: 'advance'` ferrum doesn't complain but the time advances too quickly: one `visit` and it's already the next month

any ideas?

**Comment by route** [comment]
Could you attach FERRUM_DEBUG logs as well?

**Comment by dreyks** [comment]
sure! https://gist.github.com/dreyks/ab4493c24f66d78944c789f970343bf8

**Comment by route** [comment]
There's no `Page.frameStoppedLoading` event signaling that frame is loaded. Could you try same without `setVirtualTimePolicy` and check if this event appears?

**Comment by dreyks** [comment]
yup, it does appear https://gist.github.com/dreyks/9826a7a42277798606056daa5b135121

**Comment by dreyks** [comment]
so is the issue in that chrome doesn't send `Page.frameStoppedLoading` when the timer is emulated?

**Comment by route** [comment]
I linked wrong issue, the updated comment has a link on a better one with better explanation. Bad thing is that it takes them forever to fix anything, so don't expect any real quick fix :(

**Comment by route** [comment]
@dreyks did you open an issue? closing this

**Comment by dreyks** [comment]
i didn't, needed a solution quickly so went with a workaround
